corehq/apps/smsbillables/models.py

```diff
@@ -312,26 +320,60 @@ def create(cls, message_log, api_response=None):
         country_code, national_number = get_country_code_and_national_number(phone_number)
 
         if backend_instance is None or _sms_backend_is_global(backend_instance):
-            billable.gateway_fee = SmsGatewayFee.get_by_criteria(
-                backend_api_id,
-                direction,
-                backend_instance=backend_instance,
-                country_code=country_code,
-                national_number=national_number,
-            )
-            if billable.gateway_fee is not None:
-                conversion_rate = billable.gateway_fee.currency.rate_to_default
+            currency = None
```

Comment:
May be it's more readable to group Twilio specific condition to its own function?